objetivos/leanuxcanvas.md

```diff
@@ -1,41 +1,72 @@
+-->  
 
 <table markdown="1"><tbody><tr><td markdown="1">
 
 **Problema**
 
+No estar viendo la TV o escuchando la radio el día en el que se emite el programa o el podcast o no comprar el periódico el día en el que se publica la revista. La página web también va a ofrecer la opción de consumir más contenido del que ha aparecido en el programa o en la revista. 
+
 </td><td rowspan=2 markdown="1">
 
-**Ideas de soluciones**
+**Soluciones**
+
+- Posibilidad de consumir los tres productos en cualquier momento
+- Facilitar el contenido entero del material que aparece en los medios (allí solo aparecerá una parte de todo el proceso) -> “para los más curiosos”, ”por si te lo perdiste”, ”más sobre…”
+- Posibilitar al consumidor la opción de compartir el contenido
 
 </td><td markdown="1">
 
 **Resultados**
 
+- Número de visitas
+- Interacciones
+- Cantidad de “me gusta”
+- Cantidad de veces que se ha compartido 
+- Número de visualizaciones o escuchas del contenido audio y audiovisual
+
+
 </td></tr><tr><td markdown="1">
 
 **Usuarios**
 
+Nuestro público objetivo serán jóvenes de 18 a 30 años.
+
 </td><td markdown="1">
 
-**Beneficios para los usuarios**
+**Resultados y beneficios para los usuarios**
+
+- Por falta de información sobre el tema que trataremos
+- El beneficio que tratamos de ofrecer es la mezcla de información sobre política y entretenimiento
+- Sabremos si hemos conseguido nuestro objetivo (o no) observando la interactividad de nuestro contenido en las redes sociales. Si se  han obtenido muchas interacciones, sabremos que ha sido consumido por una gran cantidad de personas
+
 
 </td></tr><tr><td markdown="1">
 
 **Hipótesis**  
 
+2. Creemos que obtendremos buenos resultados en la medición cuantitativa del número de visitas, interacciones, ”me gusta”… si hacemos una página web con usabilidad.
+3. Creemos que los jóvenes consumirán nuestro producto si hacemos una página web que se adecue a sus necesidades
+4. Creemos que los usuarios se informarán y se entretendrán si conseguimos crear contenido informativo de manera atractiva
+5. Creemos que los usuarios podrán consumir todo el contenido en cualquier lugar y a cualquier hora, además de compartirlo, si creamos un soporte web para alojar todos los contenidos y sea accesible en la web
+
 </td><td markdown="1">
 
 **¿Qué es lo más importante que tenemos que aprender primero?**
 
+2. Crear una página web con usabilidad
+3. Saber cuales son las necesidades de los jóvenes
+4. Saber crear contenido informativo y entretenido
+5. Saber crear una página web accesible para la web 
+
 </td><td markdown="1">
 
 **¿Cuál es el trabajo mínimo que tenemos que hacer para aprender lo siguiente más importante?**
 
+4 -> ENCUESTA sobre si nuestra página web es atractiva, entretenida e informativa
+
```

Comment:
Y esta caja se deriva directamente de la anterior. Empezando por las que hayáis considerado más críticas, ¿cómo podéis saber si las suposiciones se demuestran ciertas o no?

En función de esto podrá diseñarse una primera versión que poner a prueba.

Reply:
Hola @mberasategi , consideramos que con la encuesta, las lecturas acerca de nuestro público objetivo, enseñar los wireframes y preguntar a conocidos acerca de sus intereses seremos conscientes si nuestras suposiciones mencionadas anteriormente son ciertas o no.